public/editor-client/src/dynamicContent/index.ts

```diff
@@ -11,3 +15,55 @@ export const placeholders: DCHandler = async (res, rej, extraData) => {
     rej(t("Failed to load dynamic content placeholders"));
   }
 };
+
+export function getPlaceholderData(config: Config) {
+  const {
+    url,
+    actions: { placeholdersContent },
+    hash,
+    editorVersion: version
+  } = config;
+
+  return async (
+    res: Response<unknown>,
+    rej: Response<string>,
+    extra: { placeholders: Dictionary<string>; signal?: AbortSignal }
+  ) => {
+    const { placeholders, signal } = extra;
+
+    const body = new URLSearchParams({
+      hash,
+      version,
+      action: placeholdersContent
+    });
+
+    for (const [postId, placeholders_] of Object.entries(placeholders)) {
+      if (placeholders_) {
+        for (const p of placeholders_) {
+          body.append(`p[${postId}][]`, p);
+        }
```

Comment:
move all opf this logic inside the api folder same as `placeholders`
![image](https://github.com/ThemeFuse/Brizy/assets/18303258/0bc6d402-3d3f-4dd1-9672-5b76154d1189)